app/models/work.rb

```diff
@@ -98,11 +98,11 @@ class Work < ActiveRecord::Base
   # 作品のエピソード数分の空白文字列が入った配列を返す
   # Chart.jsのx軸のラベルを消すにはこれしか方法がなかったんだ…! たぶん…。
   def chart_labels
-    episodes.pluck(:id).map {''}
+    episodes.published.pluck(:id).map {''}
```

Comment:
Space missing inside {.<br>Space missing inside }.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.